components/com_joomgallery/views/detail/tmpl/default.php

```diff
@@ -197,7 +197,31 @@
         <div style="white-space:nowrap;" id="trueContainer">
 <?php   endif;
         if(count($this->images)):
-          foreach($this->images as $row): ?>
+
+			$image	= $this->get('Image');
```

Comment:
It should not be necessary to call this method here because the requested image object is already stored in `$this->image` (not to be confused with `$this->images`).